### ISSUE TITLE
Item cooldown HUD is gunked at any TPS that isn't 60

### ISSUE THREAD

**Issue by ProfanedBane** [space-wizards/space-station-14#1315]
The little circle animation that plays to let you know the cooldown of an item is completely broken if the server tps is not 60.
At 30 tps (default for our online public servers) they take around twice as long (if not longer) than they should.

This affects items such as melee weapons and Dark Souls carvings.

**\/ OLD ISSUE REPORT \/**
Expected:
- Use carving
- Epic may mays
- Carving is unusable when the countdown circle is visible
- Can use it after it's disappeared

Actual:
- Use carving
- When cooldown circle is red it's unusable
- When it starts rotating it can be used regardless of state

**Comment by Zumorica** [comment]
Can't replicate. Also, I didn't know we had those, what the--

**Comment by ProfanedBane** [comment]
Managed to repro on server, locally doesn't appear to work.

**Comment by Zumorica** [comment]
Also this was an issue for all items using cooldowns.

**Comment by SweptWasTaken** [comment]
I added these. Bright's working on unique sprites for all the carvings. Dunno about the cooldown thing, i'll see if I can fix it.

**Comment by ProfanedBane** [comment]
Updated issue with new info.
tl;dr the animations are hardcoded to assume 60 tps

**Comment by nok-ko** [comment]
Old issue, commenting with the state as of now.

There are some oddities with the cooldown HUD still, like how you can a mop stuck in this state by wielding/unwielding it and cleaning spills in rapid succession so the two cooldowns overlap:

![image](https://github.com/space-wizards/space-station-14/assets/13428215/01ca4f3c-4577-4657-8bcd-1d2cd7951c83)

I think this is to do with the "Wield" logic, though, I recall a similar bug being reported with spears as well. Notably this partially-filled progress bar sticks around even after you drop the item:

![image](https://github.com/space-wizards/space-station-14/assets/13428215/8c99932b-4c16-41d1-9e4b-2641eef8dfd9)

You can also get this to happen by wielding a spear, and dropping it while the cooldown HUD is still over the icon.

---

However, this doesn’t seem to be tickrate-dependent, at least not on a local server.
